code/LINDA/LINDA_turf_tile.dm

```diff
@@ -352,8 +352,9 @@ atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	for(var/turf/simulated/T in turf_list)
 		T.excited = 0
 		T.recently_active = 0
+		T.excited_group = null
```

Comment:
If this calls garbage_collect() afterwards, why is this necessary? garbage_collect sets the turf excited group to null as well.